src/App.js

```diff
@@ -45,6 +46,7 @@ function App() {
               <GlobalNav />
               <BetaBanner />
               <Header />
+              <LandingPage />
```

Comment:
🔧 it's this bit that needs removing and moving into `AppRoutes`. So the index path is defined [here](https://github.com/RaspberryPiFoundation/editor-ui/blob/main/src/components/AppRoutes.js#L31)